spring-graphql-api/HELP.md

```diff
@@ -1,14 +1,20 @@
+# Read Me First
+The following was discovered as part of building this project:
+
+* No Docker Compose services found. As of now, the application won't start! Please add at least one service to the `compose.yaml` file.
+
```

Comment:
This looks rather wierd, as does the same message at the bottom. I'm guessing this was added by Spring Boot?

Reply:
I did not add it